assets/src/stories-editor/helpers/index.js

```diff
@@ -1215,6 +1215,40 @@ export const maybeUpdateFontSize = ( block ) => {
 	}
 };
 
+/**
+ * Updates a block's width and height in case it doesn't use amp-fit-text and the font size has changed.
+ *
+ * @param {Object}  block                         Block object.
+ * @param {string}  block.clientId                Block client ID.
+ * @param {Object}  block.attributes              Block attributes.
+ * @param {number}  block.attributes.width        Block width in pixels.
+ * @param {number}  block.attributes.height       Block height in pixels.
+ * @param {string}  block.attributes.content      Block inner content.
+ * @param {boolean} block.attributes.ampFitText   Whether amp-fit-text should be used or not.
+ * @param {number}  block.attributes.autoFontSize Automatically determined font size for amp-fit-text blocks.
```

Comment:
Wow, great documentation of `block`

Reply:
Well I copied over from the other helper function 🙃

Reply:
Ah, haha